### ISSUE TITLE
Invalid mode specifed on running KeySeer command

### ISSUE THREAD

**Issue by siduck** [jokajak/keyseer.nvim#18]
![image](https://github.com/jokajak/keyseer.nvim/assets/59060246/f2ca37e2-c0d6-40cb-bb73-052535aa4125)

my lazy.nvim setting

```lua
  { "jokajak/keyseer.nvim", version = "*", cmd = "KeySeer", config = true },
```

**Comment by chrisgrieser** [comment]
I got the same error:
```
?  Error  12:00:12 msg_show.lua_error    KeySeer Error executing Lua callback: ....local/share/nvim/lazy/keyseer.nvim/lua/keyseer/init.lua:31: Invalid mode specified. See map
stack traceback:
	[C]: in function 'error'
	....local/share/nvim/lazy/keyseer.nvim/lua/keyseer/init.lua:31: in function <....local/share/nvim/lazy/keyseer.nvim/lua/keyseer/init.lua:23>
```

```
neovim 0.9.1 (homebrew)
neovide 0.10.4 (homebrew)
macOS 13.4.1 (M1)
```

**Comment by jokajak** [comment]
I fixed that error without having looked at the issues first. Thanks for reporting!

It's fixed in v1.3.0 (via https://github.com/jokajak/keyseer.nvim/commit/d9e59f92ca382f42b03c63d6dce43d917f314ffb)
